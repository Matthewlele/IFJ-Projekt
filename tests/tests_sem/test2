a = 56
b = "Ahoj"

c = a * b
var = chr(ahoj)
print var
