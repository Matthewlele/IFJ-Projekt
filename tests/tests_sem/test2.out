expected SEMNATIC ERR: line 5
